app/pubspec.lock

```diff
@@ -648,5 +711,5 @@ packages:
     source: hosted
     version: "2.2.1"
 sdks:
-  dart: ">=2.10.0 <2.11.0"
-  flutter: ">=1.16.0 <2.0.0"
+  dart: ">=2.10.2 <2.11.0"
```

Comment:
- [ ] Delete `pubspec.lock` and re-run `flutter pub upgrade `to regenerate it.

Reply:
Actually, one of the new additions to pubspec.yaml requires a minimum dart and flutter version. This is fine